README.md

```diff
@@ -14,4 +14,9 @@ PORT=3000
 HAPIKEY=...
 SAPIKEY=...
 STENANTALIAS=...
+// Hubspot App Properties
```

Comment:
Saasquatch is also going to have a client id/secret. Need to differentiate them with different naming conventions

Reply:
Updated!